### ISSUE TITLE
0.3.4 default language broken

### ISSUE THREAD

**Issue by funkydude** [bitbeans/SimpleDnsCrypt#54]
So I just installed the latest update and the GUI defaulted to Indonesian instead of English...

**Comment by bitbeans** [comment]
Fixed in 0.3.5

**Comment by funkydude** [comment]
@bitbeans I updated and now the language changed to German. Is this because I changed the language back to English after the previous update, or is it still broken?

**Comment by bitbeans** [comment]
hmm, i need check this later

**Comment by funkydude** [comment]
@bitbeans 
```
System Locale:             en-gb;English (United Kingdom)
Input Locale:              en-gb;English (United Kingdom)
```

**Comment by porkrinds** [comment]
Just to confirm I had the same results on two en-US system. Updated using the built-in installer, and when the program restarted, the UI was in German. Just a minor annoyance, it's two clicks to change back to English.

1x Windows 10 Enterprise x64 en-US system [Bare Metal]
1x Windows 7 Professional x64 en-US system [Hyper-V Virtualized Machine]

**Comment by bitbeans** [comment]
Solved in 0.3.6
My local config.yaml was included in the msi package :monkey_face:
